draft-ietf-dnsop-svcb-https.md

```diff
@@ -936,7 +936,7 @@ the Internet ("IN") Class {{!RFC1035}}.
 
 All the SvcParamKeys defined in {{keys}} are permitted for use in
 HTTPS RRs.  The default set of ALPN IDs is the single value "http/1.1".
```

Comment:
```suggestion
HTTPS RRs.  The default set of ALPN IDs is the single value "http/1.1".
Clients not implementing the "alpn" key SHOULD behave as if the default
ALPN ID is present and also discard records containing no-default-alpn.
```

Reply:
OK, I added a variation on that sentence.